build/chip/fuzz_test.gni

```diff
@@ -66,3 +71,68 @@ template("chip_fuzz_target") {
     }
   }
 }
+
+# Define a fuzz target for chip using pw_fuzzer and Google FuzzTest Framework.
+#
+# Fuzz generally only apply on the following environments:
+#   - linux and mac host builds when using clang
+#
+# Sample usage
+#
+# chip_pw_fuzz_target("fuzz-target-name") {
+#   test_source = [
+#      "FuzzTarget.cpp", # Fuzz target
+#   ]
+#
+#   public_deps = [
+#     "${chip_root}/src/lib/foo",         # add dependencies here
+#   ]
+# }
+#
+#
+template("chip_pw_fuzz_target") {
+  if (!defined(invoker.sources)) {
+    sources = []
+  }
+  if (!defined(invoker.public_deps)) {
+    public_deps = []
+  }
+  if (!defined(invoker.deps)) {
+    deps = []
+  }
```

Comment:
should these happen under the pw_test template? Otherwise what is their purpose (e.g. if test_source is not defined)?

Reply:
no need for them, will remove them